app/schema.graphql

```diff
@@ -142,6 +142,7 @@ type DatasetAppConfig {
   sidebarMode: SidebarMode
   modalMediaField: String
   gridMediaField: String
+  gridThumbnailFields: JSON
```

Comment:
A comment explaining what this is would be nice.

Reply:
Added to `fiftyone/core/odm/dataset.py` (which is used to generate this schema)